### DISCUSSION TITLE
Svelte or Kit?

### DISCUSSION THREAD

**Discussion by joshuavachon25** [tauri-apps/tauri#5322] [Q&A]
Are there any advantages to use Sveltekit over Svelte when creating a Tauri App? Is the SSG faster over the SPA? 

I mean, without +server pages and all SSR stuff, is there really an advantage?

**Comment by JonasKruckenberg** [comment]
Yes SSG is an advantage, on the web having pre-rendered HTML improves the First Meaningful Paint and let's your page appear faster than it may actually be. With native apps this first meaningful paint metric becomes *waaay* more important though, since users are used to their apps being snappy and proving almost instant launch feedback, not staring at a blank page for 200ms. 

So for feeling native, it's very important that you draw *something* on the screen as fast as possible, ideally pre-rendered (because that is the fastest) and using the AppShell architecture.

**Reply by theArianit** [reply]
so, we can use sveltekit and make use of SSR and SSG features of it?

**Reply by FabianLars** [reply]
yes you can use sveltekit. yes you can make use of SS**G** features. **No** you can't make use of SS**R**.

(unless you bundle a nodejs runtime yourself, or host your frontend on a remote server - both things generally not recommended of course)
